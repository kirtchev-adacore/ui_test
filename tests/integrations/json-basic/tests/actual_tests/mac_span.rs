#![deny(unused_variables)]

macro_rules! m {
    () => {{
        let x = 0; //~ unused_variables
    }};
}

fn main() {
    m!();
}
