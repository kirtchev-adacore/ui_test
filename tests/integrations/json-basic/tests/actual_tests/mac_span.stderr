error: unused variable: `x`
  --> tests/actual_tests/mac_span.rs:5:13
   |
5  |         let x = 0;
   |             ^ help: if this is intentional, prefix it with an underscore: `_x`
...
10 |     m!();
   |     ---- in this macro invocation
   |
note: the lint level is defined here
  --> tests/actual_tests/mac_span.rs:1:9
   |
1  | #![deny(unused_variables)]
   |         ^^^^^^^^^^^^^^^^
   = note: this error originates in the macro `m` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error

